memcached-operator/pkg/controller/memcached/memcached_controller.go

```diff
@@ -114,6 +122,8 @@ func (r *ReconcileMemcached) Reconcile(request reconcile.Request) (reconcile.Res
 			return reconcile.Result{}, err
 		}
 		// Deployment created successfully - return and requeue
+		// NOTE that the requeue is made with the propose to provide the deployment object for the next step to ensure the deployment size is the same as the spec
```

Comment:
Not sure what it means to "provide the deployment object for the next step", can we rephase this a bit?
```suggestion
		// NOTE: that the requeue is made with the purpose to provide the deployment object for the next step to ensure the deployment size is the same as the spec.
```

Reply:
👍 Done really tks